Src/Fido2.Models/Objects/MakeNewCredentialResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Some of these usings seem unused?

Reply:
Good catch! In fact, they are all unused. I remove them.